tradfri/tradfri-client.go

```diff
@@ -23,6 +48,9 @@ func NewTradfriClient(gatewayAddress, clientID, psk string) *Client {
 	return client
 }
 
+// Yo who decided it would be a good idea to have the deviceId be an int in all the models, but here every function wants it as a string, its stupid
```

Comment:
Good point. I'm thinking of making a v1.0 release anyway after merging this PR. I'll take a look at changing the type in the API to int before tagging.